1.9/api/access.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Cluster Access
+nav_title: Cluster Access
+menu_order: 1
+---
+
+The cluster address may be obtained using the following methods:
+
+- Log into the DC/OS GUI and copy the scheme and domain name from the browser address bar.
+- Log into the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL.
+
+
+## API ports
+
+On the master nodes, Admin Router is accessible through standard ports: `80` (http) and `443` (https, if enabled).
+
+On the agent nodes, Admin Router Agent is accessible through port `61001` (http).
+
+
+## Agent node address
+
+The IP of specific agent nodes may be obtained using the following methods:
```

Comment:
Suggest: `You can get the IP address of a specific agent node by using the following methods:`